packages/nc-gui-v2/components/dashboard/settings/UserManagement.vue

```diff
@@ -0,0 +1,287 @@
+<script setup lang="ts">
+import { useDebounce } from '@vueuse/core'
+import KebabIcon from '~icons/ic/baseline-more-vert'
+import UsersModal from './userManagement/UsersModal.vue'
+import { projectRoleTagColors } from '~/utils/userUtils'
+import MidAccountIcon from '~icons/mdi/account-outline'
+import ReloadIcon from "~icons/mdi/reload"
+import MdiEditIcon from '~icons/ic/round-edit'
+import SearchIcon from '~icons/ic/round-search'
+import { copyTextToClipboard } from '~/utils/miscUtils'
+import MdiDeleteOutlineIcon from '~icons/mdi/delete-outline'
+import EmailIcon from '~icons/eva/email-outline'
+import MdiPlusIcon from '~icons/mdi/plus'
+import MdiContentCopyIcon from '~icons/mdi/content-copy'
+import MdiEmailSendIcon from '~icons/mdi/email-arrow-right-outline'
+import RolesIcon from '~icons/mdi/drama-masks'
+import { User } from '~~/lib/types'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+const { $api, $e } = useNuxtApp()
+const { project } = useProject()
+
+let users = $ref<null | Array<User>>(null)
```

Comment:
Use the User[] notation instead of Array<User>. Both are valid but usually we use [] to type annotate arrays, especially to avoid confusion between the Array constructor of JS and the Type Array